src/WebexSDKAdapter.js

```diff
@@ -0,0 +1,13 @@
+export default class WebexSDKAdapter {
+  /**
+   * Creates a new instance of the WebexAdapter.
+   * The primary datasource model comes from the following structure:
```

Comment:
`The data source for this adapter comes from an authenticated instance of the SDK.`